test/runtests.jl

```diff
@@ -261,4 +261,14 @@ end;
     end
     @test sum(res) >= N*0.80
 end
-    
\ No newline at end of file
+
+@testset "SimpleHypergraphs connected components" begin
+    bip = LightGraphs.SimpleGraph(BipartiteView(h1))
+    cc = LightGraphs.connected_components(bip)
+    filter!.(<=(size(h1, 1)), cc)
```

Comment:
This line breaks the compatibility of the entire package with Julia 1.0.x
Instead of `<=(size(h1, 1))`   use   `x -> x <= nhv(h1)`

Reply:
In particular the test-build logs are here: https://travis-ci.org/pszufe/SimpleHypergraphs.jl and they should have "passed" status for this Pull Request

Reply:
just `<=(nhv(h1))` is enough I guess. Why is using `size` not allowed?